apps/server/src/modules/group/service/group.service.ts

```diff
@@ -19,6 +19,12 @@ export class GroupService implements AuthorizationLoaderServiceGeneric<Group> {
 		return group;
 	}
 
+	async findByExternalSource(externalId: string, systemId: EntityId): Promise<Group | null> {
```

Comment:
what does externalSource mean ? external id from user or school?

Reply:
externalSource consists of an externalId and a systemId, because you need both to make it unique. (externalIds are only unique inside 1 system and we are potentially connecting to more than 1).

As we are in the groupService here and the groupRepo is called the externalSource relates to the group entity.